src/kOS.Safe/Execution/CPU.cs

```diff
@@ -704,16 +695,17 @@ public object GetValue(object testValue, bool barewordOkay = false)
             // If it's a variable, meaning it starts with "$" but
             // does NOT have a value like $<.....>, which are special
             // flags used internally:
```

Comment:
I expect that yes, the prefix check will be obsolete.  I noted it as such with a TODO in my first run through, but as noted above the CPU changes have been put on hold so I didn't commit those changes.